corehq/apps/users/views/mobile/users.py

```diff
@@ -1685,6 +1731,60 @@ def link_connectid_user(request, domain):
         return HttpResponse()
 
 
+@require_can_edit_commcare_users
+@location_safe
+def send_connectid_invite(request, domain, user_id):
+    # Currently same as what send_confirmation_sms does
+    user = CommCareUser.get_by_user_id(user_id, domain)
+    if user.is_account_confirmed or not user.is_commcare_user() or user.domain != domain:
+        messages.error(request, "The user is already confirmed or is not a mobile user")
+        return HttpResponse(status=400)
+
+    invite_code = encrypt_account_confirmation_info(user)
+    url = absolute_reverse("confirm_connectid_user", args=[user.domain])
+    text_content = f"""
+    Link your ConnectID account to CommCare
+    username: {user.username}
+    domain: {domain}
+    invite_code: {invite_code}
+    url: {url}
+    Make a POST call to above url with invite_code and your ConnectID token
```

Comment:
I don't think we want to send the user a message telling them to make a POST, because they won't know what to do with that info. I think we want to pass the right info to the phone so it can do that automatically (or have a clickable link that will confirm it, but a link that is clickable in the sms but doesn't do the right thing is going to be confusing)

Reply:
Yeah, definitely, this is a ToDo; mobile team has given me the correct format, so I will update this.